lib/table_saw/queries/prepared_insert_statement.rb

```diff
@@ -31,7 +31,10 @@ def column_types
       end
 
       def column_names
-        TableSaw.schema_cache.columns(table_name).map(&:name).join(', ')
+        TableSaw.schema_cache.columns(table_name)
+          .map(&:name)
```

Comment:
This `map` seems unnecessary. You could extract the name in the next iteration from the column.